cmake/tools.cmake

```diff
@@ -65,10 +67,11 @@ if(USE_SANITIZER OR USE_STATIC_ANALYZER)
 endif()
 
 # enables CCACHE support through the USE_CCACHE flag possible values are: YES, NO or equivalent
+option(USE_CCACHE "enable ccache usages" ON)
```

Comment:
These settings are optional and meant to be [passed to cmake by the user](https://github.com/TheLartians/ModernCppStarter#ccache). They should not be enabled by default.

Reply:
IMHO no,  use cache and ninja to get the fasted roundtrip time